File: app/src/main/java/captain/wonjong/fontchecker/model/TextModel.java
package captain.wonjong.fontchecker.model;

import android.graphics.Color;

import androidx.lifecycle.MutableLiveData;

public class TextModel {
    public MutableLiveData<String>  text = new MutableLiveData<>();
    public MutableLiveData<Integer> bgColor = new MutableLiveData<Integer>();
    public MutableLiveData<Integer> textColor = new MutableLiveData<>();
    public MutableLiveData<Integer> textSize = new MutableLiveData<>();

    public TextModel() {
        text.setValue("");
        bgColor.setValue(Color.rgb(255,255,255));
        textColor.setValue(Color.rgb(0,0,0));
        textSize.setValue(18);
    }
}

File: app/src/main/java/com/yujin/fontchecker/model/TextModel.kt
package com.yujin.fontchecker.model

import android.graphics.Typeface
import androidx.lifecycle.MutableLiveData
import com.yujin.fontchecker.util.DEFAULT_TEXT
import com.yujin.fontchecker.util.DEFAULT_TEXT_SIZE
import com.yujin.fontchecker.util.RGB_BLACK
import com.yujin.fontchecker.util.RGB_WHITE

class TextModel(_text: String? = DEFAULT_TEXT, _textSize: Int? = DEFAULT_TEXT_SIZE) {
    val text = MutableLiveData<String>(_text)
    val bgColor = MutableLiveData<Int>(RGB_WHITE)
    val textColor = MutableLiveData<Int>(RGB_BLACK)
    val textSize = MutableLiveData<Int>(_textSize)
    val typeface = MutableLiveData<Typeface>()
}